scripts/travis_centos.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# usage: ./travis_centos.sh [:7|:8|:latest|:stream]
+#
+## -e : Make sure all errors cause the script to fail
+## -x be verbose; write what we are doing, as we do it
+set -ex
+
+# centos-stream should be pulled from quay.io
+# note: a RedHad account is required after 2021.06,
+#       see https://quay.io/repository/centos/centos?tab=tags
+if [ $1 = ":stream" ]; then
+    prefix="quay.io/centos/"
+fi
+
+if [ $1 = ":7" ]; then # for 7
+    pm="yum"
+    cmake="cmake3"
+else # for 8+, stream
+    pm="dnf"
+    cmake="cmake"
+    cmd_powertools="dnf config-manager --set-enabled powertools &&"
+fi
+
+sudo docker pull "${prefix}centos$1"                                        \
+    &&                                                                      \
```

Comment:
Can you reindent from here: would make script more readable. Thanks.

Reply:
Done.